src/components/atoms/SfCheckbox/SfCheckbox.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { shallowMount } from "@vue/test-utils";
+import SfCheckbox from "./SfCheckbox.vue";
+
+describe("SfCheckbox.vue", () => {
+  it("renders a component", () => {
+    const component = shallowMount(SfCheckbox);
+    expect(component.contains(".sf-component")).toBe(true);
```

Comment:
```suggestion
    expect(component.contains(".sf-checkbox")).toBe(true);
```

Reply:
had not edited the tests yet, now it's ready